app/posts/manage-teacher-training-applications/2020-12-09-full-time-part-time.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Filter applications by full time or part time
```

Comment:
Change the  file name to be `filter-by-full-or-part-time`

Reply:
Done